proposals/testing/0001-refactor-bug-inits.md

```diff
@@ -0,0 +1,173 @@
+# Dedicated `.bug()` functions for URLs and IDs
+
+* Proposal: [ST-0001](0001-refactor-bug-inits.md)
+* Authors: [Jonathan Grynspan](https://github.com/grynspan)
+* Status: **Implemented (Swift 6.0)**
+* Implementation: [swiftlang/swift-testing#401](https://github.com/swiftlang/swift-testing/pull/401)
+* Review: ([pitch](https://forums.swift.org/t/pitch-dedicated-bug-functions-for-urls-and-ids/71842)), ([acceptance](https://forums.swift.org/t/swt-0001-dedicated-bug-functions-for-urls-and-ids/71842/2))
+
+> [!NOTE]
+> This proposal was accepted before Swift Testing began using the Swift
+> evolution review process. Its original identifier was
+> [SWT-0001](https://github.com/swiftlang/swift-testing/blob/main/Documentation/Proposals/0001-refactor-bug-inits.md).
```

Comment:
Should we include these URLs? I was imagining we'd delete the old proposal docs and just forward to here.

Reply:
If we delete the documents from their existing locations in the swift-testing repo the links will completely break, so I am planning to instead _replace_ those files with "stubs" that mention that they have been relocated and link to the new location. These "Note" callouts are just a record that a relocation has taken place.